kiali/README.md

```diff
@@ -86,4 +105,5 @@ When you're done, you can remove the example and all its resources from the clus
 
     ```bash
     helm delete -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME}
+    kubectl -n $KYMA_KIALI_NS delete -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
     ```
```

Comment:
``````suggestion

```bash
helm delete -n $KYMA_KIALI_NS $HELM_RELEASE_NAME
kubectl -n $KYMA_KIALI_NS delete -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
```
``````
- This does not need to be a list element, as you have only one step here. Please remove the numbering and indentation. I cannot modify line 104. 
- You're using environment variables in the commands, which have already been exposed, so you must remove the curly braces from them, otherwise they won't behave as expected. 

For an alternative suggestion, see the comment below.

Reply:
❗ THIS IS AN ALTERNATIVE TO THE COMMENT ABOVE:

``````suggestion

    ```bash
    helm delete -n $KYMA_KIALI_NS $HELM_RELEASE_NAME
    kubectl -n $KYMA_KIALI_NS delete -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
    ```

2. (Optional) If you created the `$KYMA_KIALI_NS` Namespace specifically for this tutorial, remove the Namespace:
  ```bash
  kubectl delete namespace $KYMA_KIALI_NS
  ``` 
``````

Shouldn't we also remove the Namespace on cleanup? If it's only created for testing purposes.
Alternatively, you might want to leave it as step 1 and add an optional step 2 to remove the Namespace.